configure.ac

```diff
@@ -1,7 +1,7 @@
 # Package version (semver)
 define(MAJOR, 2)
 define(MINOR, 99)
-define(PATCH, 0)
+define(PATCH, 1)
```

Comment:
Version bumps are done during release (along with changelog, etc). Master branch is always x.99.0 since it's latest unreleased/unstable (hsd has a doc here: https://github.com/handshake-org/hsd/blob/master/docs/release-process.md#master-branch-and-packagejson-version)